.github/workflows/publish.yml

```diff
@@ -1,9 +1,16 @@
 name: Publish to pub.dev
 
 on:
-  push:
-    tags:
-      - 'v[0-9]+.[0-9]+.[0-9]+*'
+  workflow_dispatch:
+    inputs:
+      dryRun:
+        description: 'Do a dry run to preview instead of a real publish'
+        type: choice
+        required: true
+        default: 'true'
+        options:
+          - 'true'
+          - 'false'
```

Comment:
Can you please make this more explicit in the option names? I feel I always have to double check true/false. I updated in the SR repo:

https://github.com/amplitude/session-replay-android/blob/main/.github/workflows/release.yml#L6-L13

Reply:
Updated at https://github.com/amplitude/Amplitude-Flutter/pull/180/commits/51ea0446300c00495992d8f6e6c0b8b85a2f6f6b